test/health_check_test_case.py

```diff
@@ -198,7 +196,8 @@ def _expected_queues(self, up: bool) -> JSON:
                             'delayed': 0, 'invisible': 0, 'queued': 0
                         }
                     } if up else {
-                        'up': False, 'error': 'The specified queue does not exist for this wsdl version.'
+                        'up': False, 'error': f'The specified queue {queue_name} does not exist for '
+                                              f'this wsdl version.'
```

Comment:
The phrasing strikes me as a little redundant here.
```suggestion
                        'up': False, 'error': f'Queue {queue_name} does not exist for this wsdl version.'
```

Reply:
In the HealthController we just pass the message that was raised from the response error.
https://github.com/DataBiosphere/azul/blob/70ca8dbb0f617aac69d73bf12933c4b1dbf3e62d/src/azul/health.py#L111

This was update to match what moto 2.0.2 now returns as a response, I agree that the phrasing is off.